amenity-points.mss

```diff
@@ -4,1894 +4,1243 @@
 @shop-text: #939;
 @transportation-icon: #0092da;
 @transportation-text: #0066ff;
-@airtransport: #8461C4;
+@airtransport: #8461c4;
 @health-color: #da0092;
 @amenity-brown: #734a08;
 @man-made-icon: #555;
 @landform-color: #d08f55;
 
-@landcover-font-size: 10;
-@landcover-font-size-big: 12;
-@landcover-font-size-bigger: 15;
-@landcover-wrap-width-size: 25;
-@landcover-wrap-width-size-big: 35;
-@landcover-wrap-width-size-bigger: 45;
-@landcover-face-name: @oblique-fonts;
-
 @standard-wrap-width: 30;
 @standard-text-size: 10;
 @standard-font: @book-fonts;
 
 /* Note that .points is also used in water-features.mss */
 .points {
-  [feature = 'tourism_alpine_hut'][zoom >= 13] {
-    point-file: url('symbols/alpinehut.p.16.png');
-    point-placement: interior;
-  }
+  // markers with text (shields)
 
-  [feature = 'tourism_wilderness_hut'][zoom >= 13],
-  [feature = 'amenity_shelter'][zoom >= 16] {
-    marker-file: url('symbols/shelter.svg');
+  [feature = 'tourism_alpine_hut'][zoom >= 14],
+  [feature = 'tourism_wilderness_hut'][zoom >= 14],
+  [feature = 'amenity_shelter'][zoom >= 17],
```

Comment:
It looks like if I did, but for z13 (alpine hut, wilderness hut) and z16 (shelter) the markers do not have text, so they are in the "markers without text" category.